GSA_Adapter/GSAAdapter.cs

```diff
@@ -43,7 +43,7 @@ public partial class GSAAdapter : BHoMAdapter
 
         public GSAAdapter(string filePath = "", GSAConfig gsaConfig = null, bool active = false)
         {
-            AdapterIdName = BH.Engine.GSA.Convert.AdapterIdName;
+            AdapterIdName = Convert.AdapterIdName;
 
             Modules.Structure.ModuleLoader.LoadModules(this);
 
```

Comment:
Also, I'd probably like to see the `Private  Methods - Com Interop` below in a separate file; I'd like to keep the adapter file clean of extra things, even if these methods are used in the Constructor.
Of course this would pose the problem "where could we put those"? Maybe I'd say just another file next to this one is fine. It's not important anyway, so if you have other ideas, skip this.

Reply:
Could definitely get behind moving it out from here, but yeah.. to where.. Think just a ComCall. cs on the top level?

Reply:
I think this makes the point for the fact that, often, and as much as I don't like it, we have "utilities" methods around the adapter. We might need to formalise this into something. 
What I did in the base adapter is separating them as much as possible from the Adapter file, adding them into the PushMethods folder, for example. We might want some "SetupMethods" file for cases like this.